pandas/tests/frame/methods/test_quantile.py

```diff
@@ -59,6 +74,18 @@ def test_quantile(self, datetime_frame):
         expected = Series([3.0, 4.0], index=[0, 1], name=0.5)
         tm.assert_series_equal(result, expected)
 
+    def test_quantile_date_range(self):
+        # GH 2460
+
+        dti = pd.date_range("2016-01-01", periods=3, tz="US/Pacific")
+        ser = pd.Series(dti)
+        df = pd.DataFrame(ser)
+
+        result = df.quantile(0.5)
```

Comment:
expected is NO values?

Reply:
It has no values unless it is run with numeric_only set to false. So it would be `result = df.quantile(numeric_only=False)` instead of `result = df.quantile()`.

It probably makes more sense to change it and set numeric_only to false correct?